skimage/measure/mc_meta/createluts.py

```diff
@@ -0,0 +1,174 @@
+# -*- coding: utf-8 -*-
+
+""" Create lookup tables for the marching cubes algorithm, by parsing
+the file "LookUpTable.h". This prints a text to the stdout wich
+can then be copied to luts.py.
+
+The luts are tuples of shape and base64 encoded bytes.
+
+"""
+  
+import numpy as np
+import sys
+import base64
+
+# Get base64 encode/decode functions
+PY3 = sys.version_info[0] == 3
+if PY3:
```

Comment:
Holy moly! Please read this and fix this bit:

http://astrofrog.github.io/blog/2016/01/12/stop-writing-python-4-incompatible-code/

Also, I would use `import six`; `if six.PY2:`. That'll make it easier to track down this code once we drop PY2 support.